raster-foundry/raster-foundry
# Refactor email notification body for org/team invitation/request

## Overview

This PR refactors email notification rich HTML body for organization and team invitation or request. It enables linking to the correct URLs for users or admins to accept or decline invitations or requests.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Demo

<img width="937" alt="screen shot 2018-07-20 at 2 37 09 pm" src="https://user-images.githubusercontent.com/16109558/43019294-7043daee-8c2a-11e8-9ad3-abe89340e503.png">


### Notes

We need to add frontend support for users to request to join an organization or a team.


## Testing Instructions

 * Login as a platform member user and turn on email notifications in user profile settings. Make sure this user has a valid email address to receive emails
 * Login as a platform admin
 * Set platform email configurations at route `admin/platform/<platform id>/settings/email` and use `localhost:9091` in the "Platform Host" field for testing on dev
 * Go to organization or team user list and add the user in step 1 to this org/team
 * Check step 1 user's email inbox and make sure invitation emails are received. Check if email body copy is correct and the links work

Closes https://github.com/azavea/raster-foundry-platform/issues/389
Closes https://github.com/azavea/raster-foundry-platform/issues/415
